docs/reference/snapshot-restore/register-repository.asciidoc

```diff
@@ -153,22 +153,44 @@ unit, for example: `1GB`, `10MB`, `5KB`, `500B`. Defaults to `null` (unlimited c
 
 The URL repository (`"type": "url"`) can be used as an alternative read-only way to access data created by the shared file
 system repository. The URL specified in the `url` parameter should point to the root of the shared filesystem repository.
-The following settings are supported:
 
-[horizontal]
-`url`:: Location of the snapshots. Mandatory.
+[source,console]
+----
+PUT /_snapshot/my_read_only_url_backup
```

Comment:
This is consistent with the other examples, but I think the naming adds to the confusion. You see put...backup, when you're not putting a backup, you're specifying a backup location.

Reply:
Updated with 1866a79cf2f.